repository/t.json

```diff
@@ -3317,7 +3317,8 @@
 			"name": "Tutkain",
 			"details": "https://github.com/eerohele/tutkain",
 			"labels": ["clojure"],
-			"description": "Clojure socket REPL client",
+			"description": "An interactive Clojure editing environment for Sublime Text",
+			"labels": ["clojure", "nrepl", "repl"],
```

Comment:
You now have `labels` twice.

Reply:
That's what you get when you try to make a change like this at midnight… fixed.